ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEeaSendTransactionIntegrationTest.java

```diff
@@ -594,7 +596,9 @@ void moreThanTenNonceTooLowErrorsReturnsAnErrorToUser() {
 
     sendPostRequestAndVerifyResponse(
         request.ethSigner(sendTransaction.request(transactionBuilder.missingNonce())),
-        response.ethSigner(INTERNAL_ERROR));
+        response.ethSigner(INTERNAL_ERROR),
+        "/",
+        Optional.of(Integer.valueOf(5000)));
```

Comment:
Any reason for the manual boxing? Bit easy to read as ```Optional.of(5000)```

Reply:
Should work with just 5000 as well. Did that because the documentation said that this takes an Integer :-)
You are probably right that it is easier to read without the boxing. Will change that!